Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -76,10 +79,28 @@
                     {
                         textValue: texts.notActive,
                         remoteValue: false
-                    }]
+                    }];
             }
 
-            //Build and launch kendo grid
+
+            var self = this;
+            var showInactiveSystems = ItSystem.Settings.CatalogState.getShowInactiveSystems($window, user.id);
+            updateToggleActiveSystemsFilterBtnText();
+
+            function updateToggleActiveSystemsFilterBtnText(): string {
+                return showInactiveSystems
+                    ? "Vis aktive IT Systemer"
+                    : "Vis ikke aktive IT Systemer";
+            }
+            
+            function toggleActiveSystemsMasterFilter(): void {
+                showInactiveSystems = !showInactiveSystems;
+                ItSystem.Settings.CatalogState.setShowInactiveSystems($window, user.id, showInactiveSystems);
```

Comment:
copy paste strikes again... notice that you are changing the state of the choice of the view in the it-system-catalog with this